projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/publicize-connections.php

```diff
@@ -131,6 +135,7 @@ protected function get_connections() {
 					'connection_id'        => (string) $publicize->get_connection_id( $connection ),
 					'service_name'         => $service_name,
 					'display_name'         => $publicize->get_display_name( $service_name, $connection ),
+					'user_name'            => $publicize->get_user_name( $service_name, $connection ),
```

Comment:
You may wish to have a look at #30937 to avoid conflicts and re-use what is already added there.